README.md

```diff
@@ -40,6 +40,29 @@ Ensure packages are installed with correct version numbers by running:
   npm install --save react-dates moment@>=#.## react@>=#.## react-dom@>=#.## react-addons-shallow-compare@>=#.##
   ```
 
+#### Include component
+```
+import { SingleDatePicker } from 'react-dates';
+```
+
+#### Include CSS
+When you use Webpack with CSS loader:
+```
+import 'react-dates/lib/css/_datepicker.css';
+```
+Otherwise create a CSS file with the contents of `node_modules/react-dates/lib/css/_datepicker.css` and include it in the html `<head>` section.
```

Comment:
He means replacing `node_modules/react-dates/lib/css/_datepicker.css` with `require.resolve('react-dates/lib/css/_datepicker.css')`, the idea being that the latter doesn't assume that npm will always use the `node_modules` folder.